hw_sentiment.ipynb

```diff
@@ -2363,6 +2363,8 @@
     "\n",
     "# START COMMENT: Enter your system description in this cell.\n",
     "\n",
+    "def foo(s):\n",
```

Comment:
@SebastianHurubaru
Do we need this or was it for testing?

Reply:
@insop yes, we need to have it consistent throughout the assignments and have the students pass the test case that searches for the original system.